Sources/Data Model/FeatureVariable.swift

```diff
@@ -20,6 +20,36 @@ struct FeatureVariable: Codable, Equatable {
     var id: String
     var key: String
     var type: String
+    var subType: String?
     // datafile schema rquires this, but test has "null" value case. keep optional for FSC
     var defaultValue: String?
+    
+    enum CodingKeys: String, CodingKey {
+        case id
+        case key
+        case type
+        case subType
+        case defaultValue
+    }
+    
```

Comment:
We can drop this since all default names

Reply:
CodingKeys (Even if exactly the same as defaultNames) are required to be defined if we override `init(from decoder: Decoder)`.